geopandas/explore.py

```diff
@@ -323,6 +325,14 @@ def _colormap_helper(_cmap, n_resample=None, idx=None):
     elif not gdf.crs.equals(4326):
         gdf = gdf.to_crs(4326)
 
+    # Fields which are not JSON serializable are coerced to strings
```

Comment:
I put this here near the linearring & crs conversions. Originally had it just before producing a featurecollection, but needs to happen before categorical detection happens - if the timestamp is `column`, then it needs to already be a string to be handled properly, otherwise `bins = np.linspace(vmin, vmax, 257)[1:]` crashes.